docs/v3/getting-started/setup-with-react.md

```diff
@@ -49,15 +53,16 @@ export default function Component(): React.FC {
   return <>{account.username}</>;
 }
 ```
+
 ### State Arrays
 
 usePulse also supports **arrays** of State instances, returning values as an array that can be destructured.
 
 ```ts
 const [myState, anotherState] = usePulse([core.MY_STATE, core.ANOTHER_STATE]);
 ```
- The names of the values can be anything, though we recommend they be the camel case counter-part to the State instance name. This is completely typesafe as of version `3.1`
-  
+
+The names of the values can be anything, though we recommend they be the camel case counter-part to the State instance name. This is completely typesafe as of version `3.1`
```

Comment:
Would it be possible to put docs modifications in a separate PR, or otherwise please make a note of them in the primary PR comment?

Reply:
I thinks this changes are from jamie.. I only changed the tsconfig.json files and package.json files.. 
maybe I did something wrong by rebasing the up to date master to my master

Reply:
It could be that — I did change the base of the PR to develop rather than master, since that's where unreleased code is committed before a release is made. Could be entirely my fault! I'm sorry if so!

Reply:
sure.. next time I will pull request into develop ;D